### DISCUSSION TITLE
Was converting issues to discussions disabled? Will it be re-enabled?

### DISCUSSION THREAD

**Discussion by abgruszecki** [community/community#3114] [Discussions] [answered]
We'd (the Scala3 team) like to move issues from https://github.com/lampepfl/dotty-feature-requests to Discussions in https://github.com/lampepfl/dotty. In my private test repository, I can't seem to find a way to convert the issues. According to GH documentation this should be possible. According to #2952, this used to be possible, but apparently was disabled not too long ago. 

We'd like to try and give Discussions a go, but without being able to convert existing issues into discussions, it's not going to be doable. Can we get a confirmation that this feature was intentionally disabled? Could we know when it'll be re-enabled, if at all?

**Comment by zayfod** [comment, marked as answer]
See https://github.com/github/feedback/discussions/3105
